packages/parser/tests/events/activeMQEvent.json

```diff
@@ -1,54 +0,0 @@
-{
```

Comment:
Why did we delete all these files?

Reply:
Because we have a ton of files that are not doing nothing but are still being read for each test fixture.

I'm removing some of them in each PR to make medium size diff PRs rather than one that touches hundreds of them at once.

Reply:
> While working on this PR I also removed another batch of test event files that were not used nor referenced anywhere.

I called this out in the PR body.